templates/mastodon/index.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Output,
+  randomPassword,
+  randomString,
+  Services,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
+  const redisPassword = randomPassword();
+
+  const crypto = require("crypto");
```

Comment:
This will not work in Easypanel as the `crypto` dependecy will not exist in the frontend env. Instead, you can update the form to require the public/private keys from the user. Also, add simple description on how to obtain a key pair.

Reply:
Hello, you are already using this dependency in the "[randomPassword](https://github.com/easypanel-io/templates/blob/main/utils/schema.ts#L1)" function. Therefore, I believe there won't be any issues on the front-end.

Reply:
You're right, I forgot about it.